What you could teach sub-category page (#4157)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/explore-subjects.md

---
title: "Explore subjects you could teach"
heading: "Explore subjects"
category: Life as a teacher
image: "static/images/content/hero-images/physics_1383.jpg"

> title_paragraph: Discover what you could teach, the benefits of teaching specific subjects and how to take your next steps to become a teacher. 

Reviewer: @emmaachesongray Do you want to update this as well to reflect the new description? (Although I'm not sure where this is used!)

> title_paragraph: Discover what you could teach, the benefits of teaching specific subjects and how to take your next steps to become a teacher. 

Author: ```suggestion
title_paragraph: Discover what you could teach, what teaching different subjects is really like, and how to take your next steps to become a teacher.
```

description: |-

>   Discover what you could teach, the benefits of teaching specific subjects and how to take your next steps to become a teacher.

Reviewer: @emmaachesongray Do you want to update this as well to reflect the new description? This is the meta description so I think your new wording probably makes more sense

>   Discover what you could teach, the benefits of teaching specific subjects and how to take your next steps to become a teacher.

Author: ```suggestion
  Discover what you could teach, what teaching different subjects is really like, and how to take your next steps to become a teacher.
```
